src/mjolnir/timeparsing.cc

```diff
@@ -37,7 +37,7 @@ FormatCondition(const std::string& source, const std::regex& regex, const std::s
   return std::regex_replace(source, regex, pattern);
 }
 
-// get the dow mask from user inputed string.  try to handle most inputs
+// get the dow mask from user inputted string.  try to handle most inputs
```

Comment:
if we are going to change this it should be 
```suggestion
// get the dow mask from the user's input string.  try to handle most inputs
```

and same below. no one says inputted

Reply:
I'd rather leave grammar fixes for a future PR ... this PR is already big enough ... and I think it makes sense to separate mere typo fixing from rewriting.